src/org/jitsi/service/neomedia/RawPacket.java

```diff
@@ -1519,6 +1519,39 @@ public void setOriginalSequenceNumber(int sequenceNumber)
     }
 
     /**
+     * Sets the padding length for this RTP packet.
+     *
+     * @param len the padding length.
+     * @return the number of bytes that were written, or -1 in case of an error.
+     */
+    public int setPaddingSize(int len)
```

Comment:
Well.. we have all the other methods returning how many bytes were written.. An wouldn't it be useful if you were to write something like this:

    byte[] rtp = new byte[12];
    off += RawPacket.setVersion(rtp, 0, rtp.length);
    off += RawPacket.setPayloadType(rtp, 0, rtp.length, (byte) 100);
    ...